packages/memgraph/memgraph.1.0/opam

```diff
@@ -0,0 +1,45 @@
+opam-version: "2.0"
+synopsis: "A small library to inspect memory representation of ocaml values"
+description:
+  "Memgraph allows one to inspect an ocaml value and get a representation of its layout in memory, and helpers to dump such representation as dot files to easily print them as graphs"
+maintainer: [
+  "Guillaume Bury <guillaume.bury@gmail.com>"
+  "Armaël Guéneau <armael.gueneau@ens-lyon.org>"
+]
+authors: [
+  "Guillaume Bury <guillaume.bury@gmail.com>"
+  "Armaël Guéneau <armael.gueneau@ens-lyon.org>"
+]
+license: "MIT"
+homepage: "https://github.com/gbury/ocaml-memgraph"
+doc: "https://gbury.github.io/ocaml-memgraph/"
+bug-reports: "https://github.com/gbury/ocaml-memgraph/issues"
+depends: [
+  "dune" {>= "2.7"}
+  "ocaml" {>= "4.05.0"}
```

Comment:
There seems to be an error when building with some versions of OCaml:

```
#=== ERROR while compiling memgraph.1.0 =======================================#
# context              2.2.0~alpha~dev | linux/x86_64 | ocaml-base-compiler.4.11.2 | pinned(https://github.com/Gbury/ocaml-memgraph/releases/download/v1.0/memgraph-1.0.tbz)
# path                 ~/.opam/4.11/.opam-switch/build/memgraph.1.0
# command              ~/.opam/opam-init/hooks/sandbox.sh build dune build -p memgraph -j 31 @install
# exit-code            1
# env-file             ~/.opam/log/memgraph-9-a915ed.env
# output-file          ~/.opam/log/memgraph-9-a915ed.out
### output ###
# (cd _build/default && /home/opam/.opam/4.11/bin/ocamlopt.opt -w -40 -principal -g -I src/core/.memgraph.objs/byte -I src/core/.memgraph.objs/native -intf-suffix .ml -no-alias-deps -open Memgraph -o src/core/.memgraph.objs/native/memgraph__Repr.cmx -c -impl src/core/repr.ml)
# File "src/core/repr.ml", line 248, characters 28-41:
# 248 |       let curried_pointer = Obj.raw_field t offset in
#                                   ^^^^^^^^^^^^^
# Error: Unbound value Obj.raw_field
# Hint: Did you mean set_field?
# (cd _build/default && /home/opam/.opam/4.11/bin/ocamlc.opt -w -40 -principal -g -bin-annot -I src/core/.memgraph.objs/byte -intf-suffix .ml -no-alias-deps -open Memgraph -o src/core/.memgraph.objs/byte/memgraph__Repr.cmo -c -impl src/core/repr.ml)
# File "src/core/repr.ml", line 248, characters 28-41:
# 248 |       let curried_pointer = Obj.raw_field t offset in
#                                   ^^^^^^^^^^^^^
# Error: Unbound value Obj.raw_field
# Hint: Did you mean set_field?
```

My guess is that you need to bump this version constraint:

```suggestion
  "ocaml" {>= "4.12.0"}
```